src/playturn.cpp

```diff
@@ -189,16 +189,21 @@ turn_info::PROCESS_DATA_RESULT turn_info::process_network_data(const config& cfg
 			resources::controller->on_not_observer();
 		}
 
+		auto disp_set_team = [&](int side_index) {
```

Comment:
From what I can tell, this lambda doesn't use any local variables. Hence, don't capture them.